GliaWidgetsTests/ViewModel/Chat/ChatViewModel.Environment.Failing.swift

```diff
@@ -1,43 +1,50 @@
 @testable import GliaWidgets
 
 extension ChatViewModel.Environment {
-    static let failing = Self(
-        chatStorage: .failing,
-        fetchFile: { _, _, _ in
-            fail("\(Self.self).fetchFile")
-        },
-        sendSelectedOptionValue: { _, _ in
-            fail("\(Self.self).sendSelectedOptionValue")
-        },
-        uploadFileToEngagement: { _, _, _ in
-            fail("\(Self.self).uploadFileToEngagement")
-        },
-        fileManager: .failing,
-        data: .failing,
-        date: {
-            fail("\(Self.self).date")
-            return .mock
-        },
-        gcd: .failing,
-        localFileThumbnailQueue: .failing,
-        uiImage: .failing,
-        createFileDownload: { _, _, _ in
-            fail("\(Self.self).createFileDownload")
-            return .failing
-        },
-        loadChatMessagesFromHistory: {
-            fail("\(Self.self).loadChatMessagesFromHistory")
-            return true
-        },
-        fetchSiteConfigurations: { _ in
-            fail("\(Self.self).fetchSiteConfigurations")
-        },
-        getCurrentEngagement: { nil },
-        timerProviding: .mock,
-        uuid: {
-            fail("\(Self.self).uuid")
-            return .mock
-        },
-        uiApplication: .mock
-    )
+    static func failing(
+        fetchChatHistory: CoreSdkClient.FetchChatHistory? = nil
```

Comment:
What is the reason for adding fetching of chat history a parameter, instead of making it a part of the `Environment` struct?